tests/influence/test_influences.py

```diff
@@ -4,57 +4,105 @@
 import numpy as np
 import pytest
 
-from .conftest import create_mock_dataset
+torch = pytest.importorskip("torch")
+import torch
+import torch.nn.functional as F
+from numpy.typing import NDArray
+from torch import nn
+
+from pydvl.influence import TorchTwiceDifferentiable, compute_influences
+from pydvl.influence.general import InfluenceType, InversionMethod
+
+from .conftest import (
+    add_noise_to_linear_model,
+    linear_analytical_influence_factors,
+    linear_derivative_analytical,
+    linear_mixed_second_derivative_analytical,
+    linear_model,
+)
 
-try:
-    import torch.nn.functional as F
-    from torch.optim import Adam, lr_scheduler
 
-    from pydvl.influence.general import compute_influences
-    from pydvl.influence.linear import (
-        compute_linear_influences,
-        influences_perturbation_linear_regression_analytical,
-        influences_up_linear_regression_analytical,
+def analytical_linear_influences(
+    linear_model: Tuple[NDArray[np.float_], NDArray[np.float_]],
+    x: NDArray[np.float_],
+    y: NDArray[np.float_],
+    x_test: NDArray[np.float_],
+    y_test: NDArray[np.float_],
+    influence_type: InfluenceType = InfluenceType.Up,
+    hessian_regularization: float = 0,
+):
+    """Calculates analytically the influence of each training sample on the
+     test samples for an ordinary least squares model (Ax+b=y with quadratic
+     loss).
+
+    :param linear_model: A tuple of arrays of shapes (N, M) and N representing A
+        and b respectively.
+    :param x: An array of shape (M, K) containing the features of the
+        training set.
+    :param y: An array of shape (M, L) containing the targets of the
+        training set.
+    :param x_test: An array of shape (N, K) containing the features of the test
+        set.
+    :param y_test: An array of shape (N, L) containing the targets of the test
+        set.
+    :param influence_type: the type of the influence.
+    :param hessian_retularization: regularization value for the hessian
```

Comment:
Typo:
```suggestion
    :param hessian_regularization: regularization value for the Hessian
```
But also, the docstring just repeats the parameter name.